### DISCUSSION TITLE
objects interface: is there a way to change the text color for part of the title?

### DISCUSSION THREAD

**Discussion by frfeng** [mwaskom/seaborn#3469] [Q&A] [answered]
Hi, 

In the objects interface is there a way to change the color of part of the texts in a title, for example, the text "Adelie", "Chinstrap", and "Gentoo" in the example below? I can use math text to make a bold or italic but couldn't figure out a way to change the text color. One use case of this would be to have the text color match the mark color to avoid the need of using a legend. Thank you. 

```python
import seaborn as sns
import seaborn.objects as so

penguins = sns.load_dataset("penguins")

(
    so.Plot(penguins, x="bill_length_mm", y="bill_depth_mm")
    .add(so.Dot(), color="species")
    .label(x="Length", y="Depth", title=r"Penguin species: $\bf{Adelie}$, $\bf{Chinstrap}$, and $\bf{Gentoo}$")
)
```

**Comment by mwaskom** [comment, marked as answer]
No, I don't believe this is possible in matplotlib, sorry.
